privacy.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: privacy
+title: Privacy Notice
+---
+
+# Foreman Privacy Policy
+
+This document describes the data Foreman collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Foreman project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * Email address
+* Data that identifies you
+  * IP addresses
+  * Browser UserAgent strings
+  * Debug archives
+  * Answers to survey questions
+
+# How and why we use your data
+
+## The website
+
+The Foreman website is a static site which does not track, collect, store
+information or set cookies. There is no account for the website itself. The
+webserver (Apache) logs will contain IP addresses, timestamps, and UserAgent
+strings - these are log-rotated and deleted after 4 weeks.
+
+The webserver logs are primarily used to produce aggregated statistics about
+the community, such as popularity of different operating system packages,
+plugins, etc.
+
+* Legal basis for this data usage: Legitimate interest
+
+The website does embed YouTube videos, see [YouTube's privacy policy][YouTube]
```

Comment:
It also embeds from other sources like video.fosdem.org and slideshare.net on the media page.

Reply:
ty, will go on a link-gathering hunt. we should consider re-doing how we handle videos at some point ;)

Reply:
Fosdem doesn't seem to have a privacy policy, i doubt they capture anything at all. linked to the homepage for now